content/applications/sales/sales/invoicing/proforma.rst

```diff
@@ -52,3 +66,6 @@ invoice, to the customer.
    .. image:: proforma/pro-forma-pdf.png
       :align: center
       :alt: Sample pro-forma invoice PDF from Odoo Sales.
+
+.. seealso::
+   - :doc:`/applications/sales/sales/invoicing/invoicing_policy`
```

Comment:
No need to use a bullet point for a single seealso item, and please update the path to be relative 🙏
```suggestion
   :doc:`invoicing_policy`
```